zeroconf.py

```diff
@@ -2290,9 +2333,12 @@ def send(self, out, addr=_MDNS_ADDR, port=_MDNS_PORT):
                     USE_IP_OF_OUTGOING_INTERFACE, outgoing_ip
                 )
                 bytes_sent = s.sendto(socket_packet, 0, (addr, port))
-            except Exception:  # TODO stop catching all Exceptions
-                # on send errors, log the exception and keep going
-                self.log_exception_warning()
+            except Exception as e:  # TODO stop catching all Exceptions
+                if get_errno(e) == errno.ENETUNREACH:
+                    self.log_warning_once("Network unreachable on interface {}".format(interface))
```

Comment:
A fix to log socket write errors as simple warnings, as observed when interface becomes disconnected.